src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs

```diff
@@ -25,6 +25,7 @@ namespace OrchardCore.Localization
     /// </summary>
     public class Startup : StartupBase
     {
+        // Don't change this order to lower value because this will break the user default culture if OrchardCore.Users.Localization feature is enabled
```

Comment:
```suggestion
        // Don't change this order to a lower value because this will break the user default culture if OrchardCore.Users.Localization feature is enabled.
```